tests/func/experiments/test_experiments.py

```diff
@@ -417,7 +417,7 @@ def test_packed_args_exists(tmp_dir, scm, dvc, exp_stage, caplog):
         assert "Temporary DVC file" in caplog.text
 
 
-def test_list(tmp_dir, scm, dvc, exp_stage):
+def _prepare_experiments(tmp_dir, scm, dvc, exp_stage):
```

Comment:
We should use fixtures here. I can take it.

Reply:
there won't be a need for this if we migrate logic inside `experiments.ls` ... I would wait for that. Also, it would be a fixture that depends on other fixtures - which is fine, but makes it a bit more complicated to deal with.